src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -792,7 +792,7 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
             public ErrorDiagnostic LoopsNotSupported() => new(
                 TextSpan,
                 "BCP138",
-                "Loops are not currently supported.");
+                "For-expressions are not supported in this context. For-expressions may be used as values of resource and module declarations or as values of resource and module properties.");

```

Comment:
"...or as values of resource and module properties" - does that mean they're not supported in variable values?

Reply:
Not in this PR. Outputs will be in the next PR and variables will probably be in one of the releases right after 0.3

---
In reply to: [582075351](https://github.com/Azure/bicep/pull/1521#discussion_r582075351) [](ancestors = 582075351)